yt_dlp/extractor/bbc.py

```diff
@@ -1255,6 +1266,38 @@ def extract_all(pattern):
                 lambda s: self._parse_json(s, playlist_id, fatal=False),
                 re.findall(pattern, webpage))))
 
+        # US accessed article with single embedded video (e.g.
+        # https://www.bbc.com/news/uk-68546268)
+        next_data = traverse_obj(self._search_nextjs_data(webpage, playlist_id), (
+            'props', 'pageProps', 'page'), get_all=False)
+        video_data = traverse_obj(next_data, (
+            ..., 'contents', lambda _, v: v['type'] == 'video'), get_all=False)
+        if video_data:
+            model = traverse_obj(video_data, (
+                'model', 'blocks', lambda _, v: v['type'] == 'media',
+                'model', 'blocks', lambda _, v: v['type'] == 'mediaMetadata',
+                'model'), get_all=False)
+            if model:
+                timestamp = traverse_obj(next_data, (
+                    ..., 'contents', lambda _, v: v['type'] == 'timestamp',
+                    'model', 'timestamp', {int_or_none}, any))
+                item_id = traverse_obj(model, ('versions', 0, 'versionId', {str}))
```

Comment:
inline this in the append call

Reply:
cant since we use `self._download_media_selector(item_id)`?